pom.xml

```diff
@@ -143,6 +135,18 @@
             <version>2.0</version>
             <type>jar</type>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>matrix-project</artifactId>
```

Comment:
Why was the matrix project dependency added? The commit description makes no mention of it. I don't see the matrix plugin being used in the code. I see that it's only for testing, but I still don't see those new tests.

Reply:
That's a `test` scope dependency required by the new tests.

Reply:
Sorry, I missed to add that. I added the Matrix project test dependency because the JenkinsRule requires it and it has been removed from core since 1.561. I added the JenkinsRule in the test that verifies the configuration as code setup, according to JCasC documentation. (import and export yaml). (https://issues.jenkins-ci.org/browse/JENKINS-27826)

Reply:
Thank you for the explanation!